src/hardware/kingst-la2016/protocol.c

```diff
@@ -41,6 +41,15 @@
 #define MAX_PWM_FREQ     SR_MHZ(20)
 #define PWM_CLOCK        SR_MHZ(200)
 
+/* usb vendor class control requests to the cypress FX2 microcontroller */
+#define	CMD_EEPROM	0xa2	/* ctrl_in reads, ctrl_out writes */
+#define	CMD_FPGA_INIT	0x50	/* used before and after FPGA bitstream loading */
+#define	CMD_FPGA_SPI	0x20	/* access registers in the FPGA over SPI bus, ctrl_in reads, ctrl_out writes */
+#define	CMD_FPGA_ENABLE	0x10
+#define	CMD_BULK_RESET	0x38	/* flush FX2 usb endpoint 6 IN fifos */
+#define	CMD_BULK_START	0x30	/* begin transfer of capture data via usb endpoint 6 IN */
+#define	CMD_KAUTH	0x60	/* communicate with authentication ic U10, not used */
```

Comment:
I think these defines should be sorted by increasing hex values. I also think one space after '#define' instead of a tab is enough. While I have no strong opinion on the latter one, I think the sorting would be good to have because it makes future changes a lot easier.